guide/src/python_from_rust.md

```diff
@@ -448,13 +465,14 @@ class House(object):
         match result {
             Ok(_) => {
                 let none = py.None();
-                house.call_method1("__exit__", (&none, &none, &none)).unwrap();
-            },
+                house
+                    .call_method1("__exit__", (&none, &none, &none))
+                    .unwrap();
```

Comment:
Did you run rustfmt on the guide code samples? I didn't know that was possible 😄

Reply:
We have a Nox session `format-guide` for that. It does require a nightly toolchain though. I think @birkenfeld added this.